addons/forum-time-zones/userscript.js

```diff
@@ -0,0 +1,28 @@
+import moment from "../../libraries/moment.js";
```

Comment:
Ooooh nice! However, is Moment really needed? Aren't JavaScript Date APIs enough for this usecase?

Reply:
Moment.js supports locales and also for _example_ "Today at ..." and "Yesterday at ...".

Reply:
Fine, I don't think it's actually necessary to load such a big library for this... but at least it supports any language, I guess LOL